grammars/javascript.cson

```diff
@@ -724,6 +724,41 @@
     ]
   }
   {
+    # Math
+    'begin': '\\bMath\\b'
```

Comment:
This is what the commit currently it looks like in Atom 1.7.0:
![math-highlight-example](https://cloud.githubusercontent.com/assets/16239365/14364640/ab0769da-fd01-11e5-8e43-611f25b9b8ec.jpg)
Same behaviour as `console` and others.